setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 setup(name='code_sync',
-      version='0.0.1',
+      version='0.1.0',
```

Comment:
a new trend is to use version number using the issue year and a month or quarter i.e.: 2021.4, 2021.1. Better to judge later how updated is but deviate from the idea of a major release., update., and bugfix.

Reply:
I would prefer to use semantic versioning, as that's what I see in most python packages.

Reply:
Yes, and no big meaningful difference, was just an alternative